include/mbgl/gfx/uniform_buffer.hpp

```diff
@@ -46,37 +42,30 @@ class UniformBuffer {
     std::size_t size;
 };
 
-/// Stores a collection of uniform buffers by name
+/// Stores a collection of uniform buffers by id
 class UniformBufferArray {
 public:
-    using UniformBufferMap = mbgl::unordered_map<StringIdentity, std::shared_ptr<UniformBuffer>>;
-
     UniformBufferArray() = default;
     UniformBufferArray(UniformBufferArray&&);
     // Would need to use the virtual assignment operator
     UniformBufferArray(const UniformBufferArray&) = delete;
     virtual ~UniformBufferArray() = default;
 
     /// Number of elements
-    std::size_t size() const { return uniformBufferMap.size(); }
+    std::size_t size() const { return uniformBufferVector.size(); }
```

Comment:
Expand on the comment here to explain that this size is actually an upper limit on the number of UBOs per array now.

Reply:
Updated comment and changed the function name to `allocatedSize()`. Not sure if this is the best name, let me know if you have any other suggestions.